minecraft_ftb/Dockerfile

```diff
@@ -6,18 +6,12 @@ WORKDIR /server
 RUN wget -q https://minecraft.curseforge.com/projects/project-ozone-2-reloaded/files/2509703/download -O server.zip &&\
 	unzip server.zip &&\
 	rm server.zip &&\
-	echo "#By changing the setting below to TRUE you are indicating your agreement to our EULA (https://account.mojang.com/documents/minecraft_eula)." > eula.txt  &&\
-	echo \#$(date) >> eula.txt  &&\
-	echo "eula=true" >> eula.txt &&\
-	chmod +x ServerStart.sh &&\
-	chmod +x FTBInstall.sh &&\
-	./FTBInstall.sh
-
+	
 COPY ops.json /server/ops.json
 
 VOLUME [/server/config]
 VOLUME [/server/world]
 
 EXPOSE 25565
 
-CMD ./ServerStart.sh
+CMD ./minecraft_server.1.7.10.sh
```

Comment:
Copy the line from vanilla, replacing 'server' with 'forge'

Reply:
Can we get better init script alignment pls?